tests/test_controller_with_serializer.py

```diff
@@ -15,3 +15,16 @@ def test_controller_with_serializer():
 
     assert response.status_code == 200
     assert response.json() == expected_data
+
+
+def test_controller_with_serializer_context():
+    client = APIClient()
+    user = AuthorizedUser()
+    client.force_authenticate(user)
+
+    expected_data = {'number': 123}
+
+    response = client.get('/with-serializer/with-context/')
+
+    assert response.status_code == 200
```

Comment:
HTTPStatus.OK

Reply:
it can be changed in different PR, because of using int and HttpStatus in differentplaces